code/modules/atmospherics/machinery/pipes/smart.dm

```diff
@@ -11,16 +11,21 @@ GLOBAL_LIST_INIT(atmos_components, typecacheof(list(/obj/machinery/atmospherics)
 	construction_type = /obj/item/pipe/quaternary
 	pipe_state = "manifold4w"
 	connection_num = 0
+	var/list/connections
```

Comment:
Why is this a list?

Reply:
Why would it not be?

Reply:
because connections is a bitfield, not a list, right?

Reply:
Could I safely remove the list part of the var,  or do I replace it with something else? I'm not exactly sure how bitfields work.

Reply:
You should read this over first, they're just numbers https://tgstation13.org/wiki/Binary_flags

Can nuke the /list bit
Feel free to bug me on discord about it

Reply:
Done.